src/index.mjs

```diff
@@ -190,13 +212,71 @@ async function processCondaEnvironment(filePath) {
     }
 }
 
+// Fetch modified lines in the PR
+async function getModifiedLines(filePath) {
+    const { context } = github;
+    const baseRef = context.payload.pull_request.base.ref;
+
+    try {
+        // Fetch the base branch to ensure we have the latest state of baseRef locally
+        await execPromise(`git fetch origin ${baseRef}`);
```

Comment:
please check that baseRef is not null and provide the user with a good error message